gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.scss

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+.nx-binary-donut-chart-examples {
+  svg {
+    width: 200px;
+    height: 200px;
```

Comment:
I'd say it should have some default size, probably matching what you want it to be in the app report.  Applications would be free to change that using CSS, but there'd be no need to demonstrate it.

Reply:
> I'd say it should have some default size, probably matching what you want it to be in the app report. Applications would be free to change that using CSS, but there'd be no need to demonstrate it.

Set it to 30px by 30px as default (same as the app report size I think) https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/0b2fad02e32cf9927810950d59c52abe24ac9a2f

Reply:
> Set it to 30px by 30px as default (same as the app report size I think)

I recommend setting only the height.  It's got an inherent aspect ratio so the width will come out equal to the height automatically, and this way it's one less property for a consuming app to have to override if they want to customize it

Reply:
> > Set it to 30px by 30px as default (same as the app report size I think)
> 
> I recommend setting only the height. It's got an inherent aspect ratio so the width will come out equal to the height automatically, and this way it's one less property for a consuming app to have to override if they want to customize it

Done e8460440bc68f3dd35da44cd427a523a3659fa99